docs/advanced_topics/api/v2/usage.md

```diff
@@ -539,15 +539,15 @@ For example, `?fields=_,title` will only return the title field.
 
 ### Detail views
 
-You can retrieve a single object from the API by appending its id to the end of
+You can retrieve a single object from the API by appending its ID to the end of
```

Comment:
I think for now we should probably keep `id` lower case, this is because it's usually a reference to the model's `id` field. It's not a reference to the generic concept of identification.